docs/sphinx/formats/dataset-table.txt

```diff
@@ -398,12 +398,12 @@ to open/import a dataset in a particular format.
    * - SM Camera
      - (no extension)
      - Single file
+   * - SPC FIFO Data
```

Comment:
Thanks for spotting that I forgot to add this! We should probably add the .set extension to the formats-pages.txt too, it isn't on the supported formats table currently